frontend/src/components/Workspace/Visualize/Plot/ImagePlot.tsx

```diff
@@ -498,19 +499,24 @@ const ImagePlotChart = memo(function ImagePlotChart({
     return () => {
       window.removeEventListener("beforeunload", onCancel)
     }
-  }, [onCancel, checkStatus])
+  }, [onCancel])
 
   useEffect(() => {
+    onCancel()
```

Comment:
Does this required?

Reply:
@ReiHashimoto  Yes, I think it is necessary. onCancel has dependencies roiFilePath and WorkspaceId. When changing ROIFilePath or WorkspaceId, I think it's a good idea to cancel the pending edit

Reply:
@sanglevinh 
I get it, thank you.